### PR TITLE
Zhikai CSS Battle

### PR DESCRIPTION
**Pull request by zzhikai**
Completed CSS Battle #1 Pilot Battle

### CODE REVIEW

File: src/CSSBattle/10_CloakedSpirits/ZhangZhikai.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<div class="bgrect">
+  <div class="rectcover"></div>
```
Comment (jason-ng-zq99) [inline src/CSSBattle/10_CloakedSpirits/ZhangZhikai.html]: Consider using camelCase for naming

This is found in other files as well

Comment (jason-ng-zq99) [reply src/CSSBattle/10_CloakedSpirits/ZhangZhikai.html]: Can use VSCode's `Change All Occurrences` to help in this

File: src/CSSBattle/12_WigglyMoustache/ZhangZhikai.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    left: 230px;
+    top: 150px;
+  }
+  .innercircle {
+    background: #F5D6B4;
+    width: 60px;
+    height: 30px;
+    border-radius: 30px 30px 0 0;
+    position: fixed;
+
```
Comment (jason-ng-zq99) [inline src/CSSBattle/12_WigglyMoustache/ZhangZhikai.html]: Consider removing unnecessary line break
```suggestion
```

File: src/CSSBattle/12_WigglyMoustache/ZhangZhikai.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    left: 80px;
+    background: #D86F45;
+  }
+  .rightend {
+    width: 20px;
+    height: 10px;
+    border-radius: 0 0 10px 10px;
+    position: fixed;
+    top: 50px;
+	background: #D86F45;
```
Comment (jason-ng-zq99) [inline src/CSSBattle/12_WigglyMoustache/ZhangZhikai.html]: Consider removing unnecessary indentation
```suggestion
  background: #D86F45;
```

File: src/CSSBattle/2_Carrom/ZhangZhikai.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    background: #62374E
+  }
+  div {
+    width: 50px;
+    height: 50px;
+    position: fixed;
+    background: #fdc57b;
+  }
+  .box {
+   	left: 50px;
```
Comment (jason-ng-zq99) [inline src/CSSBattle/2_Carrom/ZhangZhikai.html:21]: Consider removing unnecessary indentation
```suggestion
   left: 50px;
```

Comment (jason-ng-zq99) [inline src/CSSBattle/2_Carrom/ZhangZhikai.html]: Consider removing unnecessary line break
```suggestion
     left: 50px;
```

File: src/CSSBattle/36_Interleaved/ZhangZhikai.html [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+<div class="bot right"></div>
+<div class="top"></div>
+<div class="top second"></div>
+<style>
+  body {
+    background: #1A4341
+  }
+  div {
+
+  }
```
Comment (jason-ng-zq99) [inline src/CSSBattle/36_Interleaved/ZhangZhikai.html]: Consider removing empty div
```suggestion
```

File: src/CSSBattle/3_PushButton/ZhangZhikai.html [hunk 6]
```diff
@@ -0,0 +1,46 @@
+<div class="rect" >
+  <div class="circle outer">
+    <div class="circle inner" />
+  </div>
+  </ div>
```
Comment (jason-ng-zq99) [inline src/CSSBattle/3_PushButton/ZhangZhikai.html]: Consider removing unnecessary indentation and spacing
```suggestion
</div>
```

File: src/CSSBattle/5_AcidRain/ZhangZhikai.html [hunk 7]
```diff
@@ -0,0 +1,41 @@
+
+<div class="top">
+  <div class="shape">
+  </div>
+  <div class="shape bottom">
+  </div>
+  
+  </div>
+  
+
```
Comment (jason-ng-zq99) [inline src/CSSBattle/5_AcidRain/ZhangZhikai.html]: Consider removing unnecessary line breaks and indentations

File: src/CSSBattle/6_MissingSlice/ZhangZhikai.html [hunk 8]
```diff
@@ -0,0 +1,42 @@
+
+<div class="shape top">
+  <div class="shape">
+  </div>
+  <div class="shape bottom">
+  </div>
+  
+  </div>
+  
+
```
Comment (jason-ng-zq99) [inline src/CSSBattle/6_MissingSlice/ZhangZhikai.html]: Consider removing unnecessary line break and indentations

File: src/CSSBattle/7_LeafyTrail/ZhangZhikai.html [hunk 9]
```diff
@@ -0,0 +1,41 @@
+
+<div class="shape left"></div>
+<div class="shape">
+</div>
+<div class="shape right">
+</div>
+
+
+
+
```
Comment (jason-ng-zq99) [inline src/CSSBattle/7_LeafyTrail/ZhangZhikai.html]: Consider removing unnecessary line breaks

File: src/CSSBattle/8_ForkingCrazy/ZhangZhikai.html [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+<div class="rect top topsec">
+</div>
+<div class="topsmallsemi topthir">
+</div>
+<div class="rect top topthir">
+</div>
+<div class="topsmallsemi topfour">
+</div>
+<div class="rect top topfour">
+</div>
```
Comment (jason-ng-zq99) [inline src/CSSBattle/8_ForkingCrazy/ZhangZhikai.html]: `<div></div>` can perhaps be on a single line to make your code neater

File: src/CSSBattle/8_ForkingCrazy/ZhangZhikai.html [hunk 11]
```diff
@@ -0,0 +1,113 @@
...
+
+<div class="rect bottom"> </div>
+
+
+
```
Comment (jason-ng-zq99) [inline src/CSSBattle/8_ForkingCrazy/ZhangZhikai.html]: Consider removing unnecessary line breaks

File: src/CSSBattle/93_GreatWall/ZhangZhikai.html [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+      height: 40px;
+      border-radius: 50%;
+      background: #F9E492;
+      position: fixed;
+      left: 140px;
+      top: 90px;
+    }
+    .bar {
+      background: #F9E492;
+      height: 16px;;
```
Comment (jason-ng-zq99) [inline src/CSSBattle/93_GreatWall/ZhangZhikai.html]: Minor typo here
```suggestion
      height: 16px;
```

File: src/CSSBattle/95_Pokeball/ZhangZhikai.html [hunk 13]
```diff
@@ -0,0 +1,74 @@
...
+<div class="botblock"></div>
+<div class="rectbehind">
+</div>
+ <div class="center"></div>
+
+
+
+
+
+
```
Comment (jason-ng-zq99) [inline src/CSSBattle/95_Pokeball/ZhangZhikai.html]: Consider removing unnecessary line breaks and moving `<div></div>` onto the same line

File: src/CSSBattle/95_Pokeball/ZhangZhikai.html [hunk 14]
```diff
@@ -0,0 +1,74 @@
...
+<style>
+  body {
+    background: #6CB3A9
+  }
+  div {
+
+  }
```
Comment (jason-ng-zq99) [inline src/CSSBattle/95_Pokeball/ZhangZhikai.html]: Consider removing unnecessary `div`
```suggestion
```

**Review by jason-ng-zq99 — APPROVED** [review]
Minor changes suggested! Other than that, 

Review 1: LGTM 😄

File: src/CSSBattle/95_Pokeball/ZhangZhikai.html (hunk 14 above)
Comment (zzhikai) [reply src/CSSBattle/95_Pokeball/ZhangZhikai.html]: 50ed92b92dcb6d7fac9a9b6965067d8e1abde559

File: src/CSSBattle/95_Pokeball/ZhangZhikai.html (hunk 13 above)
Comment (zzhikai) [reply src/CSSBattle/95_Pokeball/ZhangZhikai.html]: 50ed92b92dcb6d7fac9a9b6965067d8e1abde559

File: src/CSSBattle/93_GreatWall/ZhangZhikai.html (hunk 12 above)
Comment (zzhikai) [reply src/CSSBattle/93_GreatWall/ZhangZhikai.html]: 50ed92b92dcb6d7fac9a9b6965067d8e1abde559

File: src/CSSBattle/8_ForkingCrazy/ZhangZhikai.html (hunk 11 above)
Comment (zzhikai) [reply src/CSSBattle/8_ForkingCrazy/ZhangZhikai.html]: 50ed92b92dcb6d7fac9a9b6965067d8e1abde559

File: src/CSSBattle/8_ForkingCrazy/ZhangZhikai.html (hunk 10 above)
Comment (zzhikai) [reply src/CSSBattle/8_ForkingCrazy/ZhangZhikai.html]: 50ed92b92dcb6d7fac9a9b6965067d8e1abde559

File: src/CSSBattle/7_LeafyTrail/ZhangZhikai.html (hunk 9 above)
Comment (zzhikai) [reply src/CSSBattle/7_LeafyTrail/ZhangZhikai.html]: 50ed92b92dcb6d7fac9a9b6965067d8e1abde559

File: src/CSSBattle/6_MissingSlice/ZhangZhikai.html (hunk 8 above)
Comment (zzhikai) [reply src/CSSBattle/6_MissingSlice/ZhangZhikai.html]: 50ed92b92dcb6d7fac9a9b6965067d8e1abde559

File: src/CSSBattle/5_AcidRain/ZhangZhikai.html (hunk 7 above)
Comment (zzhikai) [reply src/CSSBattle/5_AcidRain/ZhangZhikai.html]: 50ed92b92dcb6d7fac9a9b6965067d8e1abde559

File: src/CSSBattle/3_PushButton/ZhangZhikai.html (hunk 6 above)
Comment (zzhikai) [reply src/CSSBattle/3_PushButton/ZhangZhikai.html]: 50ed92b92dcb6d7fac9a9b6965067d8e1abde559

File: src/CSSBattle/36_Interleaved/ZhangZhikai.html (hunk 5 above)
Comment (zzhikai) [reply src/CSSBattle/36_Interleaved/ZhangZhikai.html]: 50ed92b92dcb6d7fac9a9b6965067d8e1abde559

File: src/CSSBattle/2_Carrom/ZhangZhikai.html (hunk 4 above)
Comment (zzhikai) [reply src/CSSBattle/2_Carrom/ZhangZhikai.html]: 50ed92b92dcb6d7fac9a9b6965067d8e1abde559

File: src/CSSBattle/12_WigglyMoustache/ZhangZhikai.html (hunk 3 above)
Comment (zzhikai) [reply src/CSSBattle/12_WigglyMoustache/ZhangZhikai.html:72]: 50ed92b92dcb6d7fac9a9b6965067d8e1abde559

Comment (zzhikai) [reply src/CSSBattle/12_WigglyMoustache/ZhangZhikai.html]: e635fab57fb3716abf4f1629b8369570f0a3a9e1

File: src/CSSBattle/12_WigglyMoustache/ZhangZhikai.html (hunk 2 above)
Comment (zzhikai) [reply src/CSSBattle/12_WigglyMoustache/ZhangZhikai.html]: e635fab57fb3716abf4f1629b8369570f0a3a9e1

File: src/CSSBattle/10_CloakedSpirits/ZhangZhikai.html (hunk 1 above)
Comment (zzhikai) [reply src/CSSBattle/10_CloakedSpirits/ZhangZhikai.html]: 69d8fbf344b344fa08ed6bab1ef8ea25965c5b03

**Review by YongbinWang — APPROVED** [review]

**Review by YongbinWang — APPROVED** [review]
LGTM! Review 2.

**Pull request merged**